lib/driver.js

```diff
@@ -46,6 +52,11 @@ export class ChromiumDriver extends BaseDriver {
     return true;
   }
 
+  getBidiProxy() {
```

Comment:
should this be a simple getter?

Reply:
sure. i didn't think TS could handle this well but it seems it can.